assets/queries/openAPI/schema_object_using_enum_with_keyword/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	keywords := ["multipleOf", "maximum", "minimum", "exclusiveMaximum", "exclusiveMinimum", "pattern", "minLength", "maxLength", "maxItems", "minItems", "uniqueItems", "items", "required", "maxProperties", "minProperties"]
```

Comment:
I think `items` should not be on this list, since it is a Schema Object

Reply:
resolved